packages/hint-typescript-config/src/import-helpers.ts

```diff
@@ -33,7 +33,7 @@ export default class TypeScriptConfigImportHelpers implements IHint {
         const validateTslibInstalled = (evt: ScanEnd) => {
             const { resource } = evt;
 
-            const pathToTslib = path.join(process.cwd(), 'node_modules', 'tslib');
+            const pathToTslib = path.dirname(require.resolve('tslib', { paths: [resource] }));
 
             debug(`Searching "tslib" in ${pathToTslib}`);
```

Comment:
Since we won't have the full path unless `require.resolve` succeeds (and we'll be moving that below to the try/catch), I think we can still make this debug statement useful by updating it to say we're searching for tslib "relative to ${resource}".